models/gday/inst/template.params

```diff
@@ -1,258 +0,0 @@
-[git]
```

Comment:
so this got removed because write.configs writes the template file each time the model is run, right? Or did I just completely make that up?